pkg/engine/harpoon.go

```diff
@@ -775,12 +761,6 @@ func (hc *HarpoonConfig) findDiff(target *Target, method, targetPath string, com
 // Each engineMethod call now owns the prev and dest variables instead of being shared in mo
 func (hc *HarpoonConfig) EngineMethod(ctx context.Context, mo *SingleMethodObj, path string, change *object.Change) error {
 	switch mo.Method {
-	case configMethod:
-		// Only here if config has been updated by CheckForConfigUpdates.
-		// If so, update files on disk with fileTransferPodman.
-		// Updated config file is at /opt/mount/config.yaml in harpoon pod
-		// cp updated config /opt/mount/config.yaml in pod to $HOME/.harpoon/config.yaml on host
-		return fileTransferPodman(ctx, mo, defaultConfigPath, defaultHostConfigPath, nil)
```

Comment:
this filetransfer is not needed - whatever is passed to podman at /path/to/configdir:/opt/mount is where the config & backup config will be on the host (duh)

Reply:
(i give myself permission to be snarky to myself)